arcgis-ios-sdk-samples/Maps/Generate offline map (basemap by reference)/GenerateOfflineMapBasemapByReferenceViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+/// A view controller that manages the interface of the Generate Offline Map
+/// (Basemap by Reference) sample.
+class GenerateOfflineMapBasemapByReferenceViewController: UIViewController {
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set inset of overlay.
+            let padding: CGFloat = 30
```

Comment:
As far as I know, there is no way to set a view's layout margins through Interface Builder, other than extending UIView to add IB inspectable properties for the top, left, bottom, and right values.

Reply:
Set the map view's `Layout Margins` to `Fixed`:

![2019-05-06_10-51-07](https://user-images.githubusercontent.com/25374/57233630-00a11600-6fed-11e9-8953-509a12f0d191.png)

Was there a reason you wanted to use margins rather than specifying constraint constants on the four edge constraints? I imagine it's better practice in general, though I confess I haven't done that in the past (but will start now, I think).

Reply:
Ah! You've taught me something new. Thank you!